unit_test/storage.cpp

```diff
@@ -12,6 +12,8 @@
 #include <boost/test/unit_test.hpp>
 #include <boost/thread/thread.hpp>
 
+using loki::storage::Item;
```

Comment:
Need to add `namespace loki` to compile this (Or access database class ass `loki::Database`)

Reply:
fixed